Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -104,6 +104,7 @@ The gateway settings correspond to the RGW daemon settings.
 * `caBundleRef`: If specified, this is the name of the Kubernetes secret (type `opaque`) that
   contains additional custom ca-bundle to use. The secret must be in the same namespace as the Rook
   cluster. Rook will look in the secret provided at the `cabundle` key name.
+* `hostNetwork`: Whether host networking is enabled for the rgw daemon. If not set, the network settings from the cluster CR will be applied.
```

Comment:
@travisn should we be using the `network.provider` pattern from the CephCluster? I'm not sure if we might want multus or other options in the future.

Reply:
This PR initially started with the implementation of using the network.provider pattern, but it seemed like overkill and was reverted back to this simple boolean. I'd vote to keep it simple like this. In the multus case, seems like this property shouldn't set set so that the settings can be inherited from the cluster CR, and doesn't seem like we need separate multus config.